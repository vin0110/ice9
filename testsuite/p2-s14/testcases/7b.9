var ans: int;
var check: bool;
writes "Enter number of sticks: "; 
ans := read * read * 100;
check := read; 
if (check) -> 
    write "good";
fi
