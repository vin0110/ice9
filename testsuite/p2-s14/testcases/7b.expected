line 5: Semantic error: Invalid types in assignment
